deptrac.config.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Qossmic\Deptrac\Core\Layer\Collector\CollectorType;
+use Qossmic\Deptrac\Core\Layer\Collector\DirectoryCollector;
+
+return static function (Symfony\Config\DeptracConfig $config): void {
+    $config->paths(['./src']);
+
+    $config
+        ->layers('analyser')
+        ->name('analyser')
+        ->collectors()
+        ->type(CollectorType::TYPE_DIRECTORY->value)
```

Comment:
If we are to invoke the enum here, we would need to move it into the `Contract` namespace